### ISSUE TITLE
Typos in Intro to TF -> Quiz: TensorFlow Softmax

### ISSUE THREAD

**Issue by jessicayung** [udacity/sdc-issue-reports#87]
(1) First line of quiz.py
# Solution is avaialbe in the other "solution.py" tab
-> 'available' instead of 'avaialbe'.

(I think this typo appears in most quiz.py files in the Intro to TF module, likely because they were copied and pasted. Should I add issues for all of them?)

(2) Quiz Q2:
'What happens to the softmax probabilities when you multiple the logits by 10?'
-> 'multiple' should be 'multiply'.

**Comment by Brok-Bucholtz** [comment]
@jessicayung 

The typo with "avaialbe" are fixed in all the quizzes.  If you see it, you can click the "Reset Quiz" button to get the updated version of the code.  If that doesn't fix it, you should comment on the closed issue to say it's not fixed.  The issue will be reopened and worked on.

I changed "multiple" to "multiply".  Thank you for all the feedback!  You're making the degree better!

**Comment by jessicayung** [comment]
Thanks! I reloaded the page and reset the quiz and the typos are all gone. :)

**Comment by Brok-Bucholtz** [comment]
@jessicayung 

Welcome!
